### PR TITLE
Events page

### PR DESCRIPTION
**Pull request by HamadBakeel**
## **Describe what does this pull request do?**

This pull request........
created events list
created events filter
created mobile design for filters

### CODE REVIEW

**Review by adeeb-samad — APPROVED** [review]
Done

File: src/components/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            ) : (
+                <button
+                    onClick={() => handleClick(children)}
+                    className={`button ${classes}`}
+                    aria-label={ariaLabel}
+                    data-te-offcanvas-toggle
+                    data-te-target={dropDownButton.target}
+                    aria-controls={dropDownButton.controls}
+                    data-te-ripple-init
+                    data-te-ripple-color='dark'
```
Comment (iiAbady) [inline src/components/Button.jsx:29]: Awesome work @HamadBakeel! 

here I think its better to have it as light to match our design.

Comment (HamadBakeel) [reply src/components/Button.jsx:29]: as light ?

**Review by SufyanCS — APPROVED** [review]
Great work!

**Pull request merged**